docs/examples/en/math/Lut.html

```diff
@@ -11,135 +11,149 @@
-		um — The unit of measurements of the legend.<br />
-		ticks — The number of ticks to be displayed.<br />
-		decimal — The number of decimals to be used for legend values.<br />
-		notation — Legend notation: standard (default) or scientific.<br />
-		callback — An optional callback to be used to format the legend labels.<br />
+			If object is a Lut, copy into this.
 		</p>
+
+		<h3>[method:this setColorMap]( [param:String colormapName], [param:Integer numberOfColors] )</h3>
 		<p>
-		Sets the labels of the legend of this Lut.
+			colormapName - The name of colormap.<br />
+			numberOfColors - Number of colors.
+		</p>
+		<p>
+			Set colormap for this lut.
 		</p>
 
 		<h3>[method:Lut setMin]( [param:Float minV] )</h3>
 		<p>
-		minV — The minimum value to be represented with the lookup table.<br />
+			minV — The minimum value to be represented with the lookup table.
 		</p>
 		<p>
-		Sets this Lut with the minimum value to be represented.
+			Sets this Lut with the minimum value to be represented.
 		</p>
 
 		<h3>[method:Lut setMax]( [param:Float maxV] )</h3>
 		<p>
-		maxV — The maximum value to be represented with the lookup table.<br />
+			maxV — The maximum value to be represented with the lookup table.
 		</p>
 		<p>
-		Sets this Lut with the maximum value to be represented.
+			Sets this Lut with the maximum value to be represented.
 		</p>
 
-		<h3>[method:Lut changeNumberOfColors]( [param:Float numberOfColors] )</h3>
+		<h3>[method:HTMLCanvasElement updateCanvas]( [param:HTMLCanvasElement canvas] )</h3>
 		<p>
-		numberOfColors — The number of colors to be used to represent the data array.<br />
+			canvas — Canvas to be updated 
 		</p>
 		<p>
-		Sets this Lut with the number of colors to be used.
+			Draw colors onto given canvas.
 		</p>
 
-		<h3>[method:Lut changeColorMap]( [param:Float colorMap] )</h3>
+		<h2>Statics</h2>
+
+		<h3>[method:null AddColorMap]( [param:String colormapName], [param:Array arrayOfColorStops] )</h3>
 		<p>
-		colorMap — The name of the color map to be used to represent the data array.<br />
+			colormapName — The name of colormap.<br />
+			arrayOfColorStops — Array of [stop, color] pairs.
 		</p>
 		<p>
-		Sets this Lut with the colormap to be used.
+			Insert a new colormap into the set of available colormaps.
 		</p>
 
-		<h3>[method:Lut addColorMap]( colorMapName, arrayOfColors )</h3>
-		<p>
-		Insert a new color map into the set of available color maps.
-		</p>
+		<h2>Examples</h2>
 
-		<h3>[method:Lut getColor]( value ) [param:Lut this]</h3>
-		<p>
-		value -- the data value to be displayed as a color.
-		</p>
 		<p>
-		Returns a [page:Color].
+			[example:webgl_geometry_colors_lookuptable Geometry / Colors / LookupTable]
 		</p>
 
 		<h2>Source</h2>
 
 		<p>
-			[link:https://github.com/mrdoob/three.js/blob/master/examples/jsm/math/[path].js examples/jsm/math/[path].js]
+			[link:https://github.com/mrdoob/three.js/blob/master/examples/jsm/math/Lut.js examples/jsm/math/Lut.js]
```

Comment:
Hmm, this needs to be fixed somewhere else. Better to leave it as it is for this PR.

Reply:
https://github.com/mrdoob/three.js/blob/4d875bbd8f05be5eae13366b06c03e7aaae79cec/docs/page.js#L45